todoapp.js

```diff
@@ -0,0 +1,19 @@
+document.addEventListener("DOMContentLoaded", () => {
+  let sub = document.querySelector("#submit");
+  let list = document.getElementById("list");
+
+  let input = document.getElementById("things");
+
+  sub.addEventListener("click", event => {
+    let create_li = document.createElement("li");
+    if (input.value === "") {
+      // let create_par = document.createElement("p");
+      error.innerHTML = "<p>Input somethin'</p>";
+    } else {
+      create_li.innerText = input.value;
+      event.target.parentNode.appendChild(create_li, event.target);
+
+      error.innerHTML = "<p></p>";
+    }
+  });
+});
```

Comment:
The only part of the assignment missing is:

- When the user clicks on one of the li items, the item should be crossed out, indicating that that to-do is complete.

You'll want to add an event listener to your `list` variable that changes the style of the targeted element. 
```
  list.addEventListener("click", event => {
    event.target.style.textDecoration = "line-through"
  })
```
Let me know if you have any questions!

Nice job with this assignment 👍